<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_2</class>
 <widget class="QMainWindow" name="MainWindow_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1908</width>
    <height>886</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border:none;
background-color:transparent;
padding:0;
margin:0;
color:#fff;
}

#centralwidget{
background-color:#1f232a;
}










#leftmenusubcontainer{
background-color:#16191d;
}

#leftmenusubcontainer QPushButton{

padding:5px 10px;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
}

#headercontainer{
background-color:#2c313c;
}

#frame_6{
padding:5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="mainbodycontainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="mainbodycontainer_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>0</y>
            <width>1931</width>
            <height>81</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
background-color:#131519;
}</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>750</x>
             <y>10</y>
             <width>431</width>
             <height>51</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string> SWARMULATOR</string>
           </property>
          </widget>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>91</width>
             <height>1291</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
background-color:#131519;
}</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="browse_2">
          <property name="geometry">
           <rect>
            <x>870</x>
            <y>810</y>
            <width>201</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  border: 1.5px solid #2707de;
  border-radius: 5px;  /* Optional: Add rounded corners */
  padding: 1px 5
px;  /* Optional: Adjust padding for button text */
}</string>
          </property>
          <property name="text">
           <string>previous</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>71</width>
            <height>1081</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
background-color:#131519;
}</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>1850</x>
            <y>0</y>
            <width>71</width>
            <height>1101</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{
background-color:#131519;
}</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>420</y>
            <width>281</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
                border: 2px solid #2707de; /* Adjust border width as needed */
                border-radius: 4px; /* Optional: Add rounded corners */
            }

            QComboBox::item:selected {
                background-color: #淡蓝色; /* Example custom selection color (light blue) */
            }</string>
          </property>
          <item>
           <property name="text">
            <string>  Whale Optimization Algorithm (WOA)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>  Coati Optimization Algorithm(COA)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>  Grey Wolf Optimization(GWO)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>  Artificial Bee Colony(ABC)</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>330</y>
            <width>461</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select The Swarm Intelligent Algorithms</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_5" native="true">
          <property name="geometry">
           <rect>
            <x>690</x>
            <y>220</y>
            <width>681</width>
            <height>431</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_5 {
                border: 2px solid #2707de; /* Adjust border width as needed */
                border-radius: 4px; /* Optional: Add rounded corners */
            }

          </string>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>411</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Selected Algorithm</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>115</y>
             <width>301</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Accuracy Train</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>189</y>
             <width>311</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Precision Train</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>270</y>
             <width>291</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Recall Train</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>110</y>
             <width>261</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Accuracy Test</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>190</y>
             <width>271</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Precision Test</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>270</y>
             <width>271</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Recall Test</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>349</y>
             <width>641</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Selected Features:</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="browse_3">
          <property name="geometry">
           <rect>
            <x>1150</x>
            <y>810</y>
            <width>201</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  border: 1.5px solid #2707de;
  border-radius: 5px;  /* Optional: Add rounded corners */
  padding: 1px 5
px;  /* Optional: Adjust padding for button text */
}</string>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1908</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
